web/src/containers/students/Student.js

```diff
@@ -1,16 +1,108 @@
-import React from "react";
 import PropTypes from "prop-types";
+import React, {useEffect,useState } from 'react';
+import './Student.css';
+import ProgressBar from "./Progress";
+const axios = require('axios');
+import Box from '@material-ui/core/Box';
+import{
+  Home,
+  Euro,
+  FavoriteBorder,
+  Create,
+  School,
+  Room
+} from '@material-ui/icons';
+import {
+  Typography,
+  TextField,
+  Button,
+  Grid
+} from "@material-ui/core";
 
-const Student = (props) => {
-  return <div>Student #{props.match.params.id}</div>;
-};
+
+function Student() {
+    let [user,setUser]= useState({});
+    useEffect(()=>{
+      axios.get("https://my-json-server.typicode.com/pferreira101/tuichain_faker/student").then(response=>{setUser(response.data)});
+    },[]);
+    return (
+      <Box className="student" component="span" m={1}>
+       <Grid container spacing={2} className="container">
+        <Grid className="left-cont" item xs={12} md={6}>
+        <Box>
+        <img src={user.photo}/>
+        </Box>
+        </Grid>
+        <Grid className="right-cont" item xs={12} md={6}>
+        <Box className="right">
```

Comment:
está alinhado à esquerda, quando se mete na versão de telemóvel fica mal

Reply:
Tu é que me tinhas dito para nao alinhar ao centro que ficava muito espaço em branco, ponho como então?

Reply:
centrar na versão mobile

Reply:
mas dá para centrar so numa versão?

Reply:
https://material-ui.com/pt/components/use-media-query/#using-material-uis-breakpoint-helpers
ou
https://material-ui.com/pt/customization/breakpoints/#withwidth
(o segundo diz que vai ser descontinuado)

Reply:
essa cena do use media query não me parece dar par fazer a cena que eu quero, acho que é mais para ver os limites para cada dispositivo

Reply:
mas isso parece chinês por isso posso nao ter entendido bem

Reply:
e pronto. vês se o limite é 'xs' e se for centras o texto

Reply:
ja funcionou